src/Api/Movies/MoviesApi.tsx

```diff
@@ -0,0 +1,51 @@
+const serverUrl = "http://18.60.129.53/";
```

Comment:
You are duplicating the `serverUrl` here. 

Also generally the files that provide access to APIs are called **Services**. So it would make sense to call it `MovieService`

Reply:
Created an .env filed and initialized the serverUrl as env variable and imported them when required.